php/lumen/composer.json

```diff
@@ -1,6 +1,7 @@
 {
     "require": {
-        "laravel/lumen-framework": "5.7.7"
+        "laravel/lumen-framework": "5.8.0",
+        "vlucas/phpdotenv": "3.0.0"
```

Comment:
This is the wrong version.

Reply:
There is a 5.8 version available https://packagist.org/packages/laravel/lumen-framework

Reply:
I am referring to 3.0.0. It should be 3.3.0 or higher. Preferably 3.3.3.